ccips/ccip-011/ccip-011-citycoins-stacked-tokens-voting.md

```diff
@@ -0,0 +1,97 @@
+# CCIP-011
+
+## Preamble
+
+| CCIP Number   | 011                                   |
+| ------------- | ------------------------------------- |
+| Title         | CityCoins Stacked Tokens Voting       |
+| Author(s)     | Jason Schrader jason@joinfreehold.com |
+| Consideration | Technical, Governance                 |
+| Type          | Standard                              |
+| Status        | Activation-In-Progress                |
+| Created       | 2022-03-25                            |
+| License       | BSD-2-Clause                          |
+| Replaces      | CCIP-006                              |
+
+## Introduction
+
+The Stacks blockchain is a Layer 1 blockchain connected to Bitcoin, in which miners spend Bitcoin to bid for and win a fixed amount of Stacks tokens. Stackers have the option to lock up Stacks tokens for a specified amount of time, and in turn, receive a portion of the Bitcoin spent by miners proportionate to the amount Stacked[^1].
+
+Taking this concept a level further, a new token (a "CityCoin") is created on the Stacks blockchain, following the SIP-010 fungible token standard[^2], such that CityCoins can be mined and Stacked per the methods stated above except a portion of the miner's bid would be redirected to the city's wallet overseen by a trusted third party custodian.
+
+CityCoins leverage similar properties from the Proof of Transfer (PoX) consensus mechanism[^3] of the Stacks blockchain, programmed through a smart contract in Clarity.
+
+This proposal describes the on-chain vote method used to ratify CCIP-008, CCIP-009, and CCIP-010.
+
+## Specification
+
+The voting contract is designed similarly to the SIP-012 voting process[^4], in which the user's vote is equal to the amount of tokens stacked across a defined set of reward cycles.
+
+### Voting Actions
+
+Through the `vote-on-proposal` function, a principal can submit their vote for a defined proposal and the contract will query the core contract for their stacked tokens, then update and track the vote totals for the proposal and the individual principal.
+
+Within the vote contract are vailables to track:
```

Comment:
```suggestion
Within the vote contract are variables to track:
```
typo?

Reply:
Good catch! Fixed in https://github.com/citycoins/governance/pull/5/commits/8696a04bafa4ef9a2449ff857875584b2b2b9b23